framework/encode/d3d12_capture_manager.cpp

```diff
@@ -466,20 +464,6 @@ bool D3D12CaptureManager::IsUploadResource(D3D12_HEAP_TYPE type, D3D12_CPU_PAGE_
     return false;
 }
 
-uint64_t D3D12CaptureManager::GetResourceSizeInBytes(ID3D12Device_Wrapper*      device_wrapper,
-                                                     const D3D12_RESOURCE_DESC* desc)
-{
-    auto device = device_wrapper->GetWrappedObjectAs<ID3D12Device>();
-    return graphics::dx12::GetResourceSizeInBytes(device, desc);
-}
-
-uint64_t D3D12CaptureManager::GetResourceSizeInBytes(ID3D12Device8_Wrapper*      device_wrapper,
-                                                     const D3D12_RESOURCE_DESC1* desc)
-{
-    auto device = device_wrapper->GetWrappedObjectAs<ID3D12Device8>();
-    return graphics::dx12::GetResourceSizeInBytes(device, desc);
```

Comment:
If we have meta data block for GetResourceAllocationInfo(), so don't need to call this function while replay.